src/main/groovy/edu/oregonstate/mist/cardsapi/resources/CardsResource.groovy

```diff
@@ -65,4 +80,73 @@ class CardsResource extends Resource {
         }
 
     }
+
+    //Get cards by parameters
+    @GET
+    @Path ('')
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
Since this is used at the class level, you don't need to repeat it for each method